decimal.go

```diff
@@ -52,6 +51,10 @@ var DivisionPrecision = 16
 // silently lose precision.
 var MarshalJSONWithoutQuotes = false
 
+// TrimTrailingZeroes should be set to true if you want to the decimal string
+// representation to be in a format where trailing zeroes are discarded.
+var TrimTrailingZeroes = false
```

Comment:
Also, since it was `true` before, wouldn't it be better to leave it that way, unless this "public" package is actually tailored for Aardvark only?

Reply:
While I made it to default to true, I am now starting to debate whether I like this decision.

I am wondering whether we are supposed to adapt this to `shopsprings`defaults; I think this is a separate project with different ideas and goals. And I think that trimming zeroes should be disabled by default. Thoughts?

Reply:
Hmm, agreed, however I think the package could be better documented, could we add these options to the main readme file?

Reply:
Will do.